mne/data/coil_def_Elekta.dat

```diff
@@ -39,6 +39,12 @@
 #			2	accurate
 #
 #
+1   2000    2   1  0.000e+00  0.000e+00	"Point magnetometer, x-normal"
```

Comment:
Sorry, I defined them that way on my own. These aren't actually ported from Elekta's coil definitions or anything. I just added them to the `coil_def_elekta.dat` for simplicity.